storage/statedb/database.go

```diff
@@ -1103,3 +1104,22 @@ func (db *Database) UpdateMetricNodes() {
 		db.trieNodeCache.UpdateStats()
 	}
 }
+
+// SaveTrieNodeCacheToFile saves the current cached trie nodes to file to reuse when the node restarts
+func (db *Database) SaveTrieNodeCacheToFile(filePath string) {
+	if db.trieNodeCache == nil {
+		return
+	}
+	filePath = filePath + "/trie_node_cache_backup"
+	start := time.Now()
+	go func() {
```

Comment:
Dosen't it need timeout or something?

Reply:
@jeongkyun-oh If we want something like timeout, I think we should change the code inside fastcache.
Do you have a way to do this without changing fastcache package?

Reply:
Oh, not really. I checked your PR https://github.com/VictoriaMetrics/fastcache/pull/39.